src/katatraining/fibonacci/Fibonacci.java

```diff
@@ -0,0 +1,13 @@
+package katatraining.fibonacci;
+
+public class Fibonacci {
+    public static long fib(int n) {
+        int[] x = new int[n + 2];
+        x[0] = 0;
+        x[1] = 1;
+        for (int i = 2; i <= n; i++) {
+            x[i] = x[i - 1] + x[i - 2];
+        }
+        return x[n];
```

Comment:
Yes, sure, your fix is simple. But I think we don't need to create an array at all before your aswer equals incomming parameter. Let's see:
n = 0, x[n] = 0
n = 1, x[n] = 1

so, maybe, if n <2, you should return n? It would safe our memory and running time.